crates/sncast/tests/e2e/call.rs

```diff
@@ -160,11 +169,12 @@ fn test_wrong_calldata() {
     let snapbox = runner(&args);
     let output = snapbox.assert().success();
 
-    assert_stderr_contains(
+    // 0x496e70757420746f6f206c6f6e6720666f7220617267756d656e7473 is "Input too long for arguments"
+    assert_stdout_contains(
         output,
         indoc! {r"
         command: call
-        error: An error occurred [..]Execution failed[..]Input too long for arguments[..]
+        response: [0x496e70757420746f6f206c6f6e6720666f7220617267756d656e7473]
```

Comment:
Why isn't it decoded now?

Reply:
It's coming from devnet (which gets that from blockifier). Previously it was returned as error, now it's a response (not sure why, it's not because of our codebase). Create #3107 for that.